index.html

```diff
@@ -233,16 +232,16 @@ <h2>Survey</h2>
         </a>
         2019
         </p>
-        <img src="sc-logo.png" sizes="70px" srcset="sc-logo@1.5x.png 105w", "sc-logo@2x.png 140w", "sc-logo@3x.png 210w", "sc-logo@4x.png 280x" alt="The Sault College Logo">
+        <img src="sc-logo.png" sizes="70px" srcset="sc-logo@1.5x.png 105w, sc-logo@2x.png 140w, sc-logo@3x.png 210w, sc-logo@4x.png 280w" alt="The Sault College Logo">
     </footer>
     <div class="back-to-top">
     <a href="#top">  ◤ </a>
     </div>
 
 </div>
-<script src="styles/baguetteBox.js">
+<script>
     baguetteBox.run('.gallery-thumbnails', {
-    filter: /picsum\.photos\/id/i
+        filter: /picsum\.photos\/id/i
```

Comment:
You can't use both the src attribute and put JavaScript inside the element.  Do one or the other